core/src/main/scala/io/finch/package.scala

```diff
@@ -1,13 +1,16 @@
 package io
 
 import cats.effect.IO
-
+import cats.~>
 /**
  * This is a root package of the Finch library, which provides an immutable layer of functions and
  * types atop of Finagle for writing lightweight HTTP services.
- */
+  */
 package object finch extends Outputs with ValidationRules {
 
+  trait ToEffect[F[_], E[_]] extends (F ~> E)
```

Comment:
Can this live in it's own file? Also, I'd probably lobby for `io.finch.internal` package for this.

Reply:
Absolutely can do, but I don't think it's for internal, the typeclass is intent to let user implement for their own type if they are not using *Future, my intention is keep them in `io.finch.implicits._` just like cats, doobie, http4s etc
but to address @sergeykolbasov and @rpless 's concern for user confusing about import implicits
this is a good place when user simply just `import io.finch._` and get all those implicits

Reply:
Sounds good.